database/schema.sql

```diff
@@ -131,12 +131,13 @@ CREATE UNIQUE INDEX manifestsig_manifestid_investigatorid_idx ON manifestsig USI
 
 CREATE SEQUENCE loaders_id_seq START 1;
 CREATE TABLE loaders (
-	id         numeric NOT NULL DEFAULT nextval('loaders_id_seq'),
-	loadername character varying(256) NOT NULL,
-	loaderkey  character varying(64) NOT NULL,
-	name       character varying(2048),
-	env        json,
-	tags       json
+	id            numeric NOT NULL DEFAULT nextval('loaders_id_seq'),
+	loadername    character varying(256) NOT NULL,
+	loaderkey     character varying(64) NOT NULL,
+	name          character varying(2048),
+	env           json,
+	tags          json,
+    	lastused      timestamp with time zone NOT NULL
```

Comment:
the last time the loader entry was used

so more specifically the last time we saw a successful API call for a given loader